Build failure in PyTorch Windows CI doesn't terminate early enough

In this log: https://ci.pytorch.org/jenkins/job/pytorch-builds/job/pytorch-win-ws2016-cuda9-cudnn7-py3-build/1241/consoleFull

The true failure is:

```
C:\Jenkins\workspace\pytorch-builds\pytorch-win-ws2016-cuda9-cudnn7-py3-build\aten\src\ATen\native\cudnn\Conv.cpp(1207): error C2398: Element '2': conversion from 'int64_t' to '::size_t' requires a narrowing conversion
```

However, there are many many pages of scrollback after this, and eventually the thing that terminates the build is:

```
ninja: error: 'torch/lib/tmp_install/share/ATen/Declarations.yaml', needed by 'torch/csrc/autograd/generated/Functions.cpp', missing and no known rule to make it
Command '['C:\\Jenkins\\workspace\\pytorch-builds\\pytorch-win-ws2016-cuda9-cudnn7-py3-build\\Miniconda3\\lib\\site-packages\\ninja\\data\\bin\\ninja', '-f', 'build\\build.global.ninja']' returned non-zero exit status 1.
-- Building with NumPy bindings
-- Detected cuDNN at C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v9.0\lib\x64\cudnn.lib, C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v9.0\include
-- Detected CUDA at C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v9.0
-- Not using NCCL
-- Building without distributed package
Build step 'Execute shell' marked build as failure
Finished: FAILURE
```

This is not very user friendly.

--- a/.jenkins/pytorch/win-build.sh
+++ b/.jenkins/pytorch/win-build.sh
@@ -47,7 +47,7 @@ mkdir %CD%\\tmp_bin
 if "%REBUILD%"=="" (
   :check_sccache
   %CD%\\tmp_bin\\sccache.exe --show-stats || (
-    taskkill /im sccache.exe /f /t || set ERRORLEVEL=0
+    taskkill /im sccache.exe /f /t || ver > nul
     del %CD%\\tmp_bin\\sccache.exe
     aws s3 cp s3://ossci-windows/sccache.exe %CD%\\tmp_bin\\sccache.exe
     goto :check_sccache
@@ -95,7 +95,8 @@ if "%REBUILD%"=="" (
   set NO_CUDA=1
   python setup.py install
 )
-if %errorlevel% neq 0 exit /b %errorlevel%
+if errorlevel 1 exit /b 1
+if not errorlevel 0 exit /b 1
 if "%REBUILD%"=="" (
   sccache --show-stats
   sccache --zero-stats

--- a/tools/build_pytorch_libs.bat
+++ b/tools/build_pytorch_libs.bat
@@ -162,7 +162,8 @@ goto:eof
                   -DCMAKE_BUILD_TYPE=%BUILD_TYPE%
 
   %MAKE_COMMAND%
-  IF NOT %ERRORLEVEL%==0 exit 1
+  IF ERRORLEVEL 1 exit 1
+  IF NOT ERRORLEVEL 0 exit 1
   cd ../..
   @endlocal
 
@@ -198,7 +199,8 @@ goto:eof
                   -DWITH_ROCM=%WITH_ROCM%
 
   %MAKE_COMMAND%
-  IF NOT %ERRORLEVEL%==0 exit 1
+  IF ERRORLEVEL 1 exit 1
+  IF NOT ERRORLEVEL 0 exit 1
   cd ..
   @endlocal
 

